charts/workflows/templates/pod-securitycontext-clusterpolicy.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: pod-securitycontext
+spec:
+  rules:
+    - name: pod-securitycontext
+      match:
+        resources:
+          kinds:
+            - Pod
+          namespaceSelector:
+            matchLabels:
+              app.kubernetes.io/managed-by: sessionspaces
+      context:
+        - name: values
+          configMap:
+            name: sessionspaces
+            namespace: "{{ `{{request.object.metadata.namespace}}` }}"
+      mutate:
+        patchStrategicMerge:
+          spec:
+            securityContext:
+              runAsGroup: "{{ `{{values.data.gid | parse_json(@).to_number(@)}}` }}"
```

Comment:
I think we also want to set the `runAsUser` to a value set in the `values.yaml`. We may also want to reject any pods with `allowPrivilegeEscalation` set to `true`